assets/js/modules/analytics/datastore/properties.js

```diff
@@ -358,6 +364,9 @@ export const selectors = {
 	 * @return {?Array.<Object>} An array of Analytics properties; `undefined` if not loaded.
 	 */
 	getProperties( state, accountID ) {
+		if ( typeof accountID === 'undefined' ) {
+			return undefined;
+		}
```

Comment:
Why is this here? Passing an empty `accountID` should instead run into the `invariant` error.

Reply:
It's because we have to select everything at once without conditionals due to the way hooks work, so if something isn't set yet, it will throw invariant errors. The solution is to allow `undefined` to pass though since it is coming from another selector itself which may not be resolved yet.

Reply:
My first thought is that I'd prefer to catch `undefined` values before calling this selector then - could we do that instead?

If someone calls this like `select( 'modules/analytics' ).getProperties()`, it should throw an error, but the way it behaves now it doesn't, which is problematic.